packages/siopv2-oid4vp-op-auth/src/session/OpSession.ts

```diff
@@ -221,12 +220,13 @@ export class OpSession {
     if (!resolveOpts.subjectSyntaxTypesSupported || resolveOpts.subjectSyntaxTypesSupported.length === 0) {
       resolveOpts.subjectSyntaxTypesSupported = await this.getSupportedDIDMethods(true)
     }
+    //todo: populate with the right verification params. In did-auth-siop we don't have any test that actually passes this parameter
+    // and furthermore, it receives a couple of parameters that don't make that much sense for OP. like presentationVerificationCallback and revocationOpts
     const verification: Verification = {
-      mode: VerificationMode.INTERNAL,
-      checkLinkedDomain: CheckLinkedDomain.IF_PRESENT,
-      resolveOpts,
+      // mode: VerificationMode.INTERNAL,
```

Comment:
Why is this disabled, as the default is an opt in anyway

Reply:
the verification object has been changed:
```
export interface Verification {
  presentationVerificationCallback?: PresentationVerificationCallback
  revocationOpts?: RevocationOpts
  replayRegistry?: IRPSessionManager
}
```
and the todo that I've added is talking about populating it with the correct values